### DISCUSSION TITLE
THEME HOSPITAL PS1 NEW REQUEST

### DISCUSSION THREAD

**Discussion by antoniolgruiz8** [garungorp/MouseInjectorDolphinDuck#16] [General]
Could you please include Theme Hospital as a new game supported for the Mouse Injector? Thank you so much in advance!

**Comment by Akelaphobia** [comment]
There's already a requests thread, use that instead of making a new thread. https://github.com/garungorp/MouseInjectorDolphinDuck/discussions/1

**Reply by antoniolgruiz8** [reply]
Thank you for letting me know!! Just posted on that thread
